pkg/agent/runner.go

```diff
@@ -1551,6 +1570,35 @@ func (r *Runner) doVMUpdate(
 	return &target, nil
 }
 
+func (r *Runner) recordResourceChange(current, target api.Resources, metrics resourceChangePair) {
+	abs := current.AbsDiff(target)
+
+	// Add CPU
+	if abs.VCPU != 0 {
+		direction := "up"
+		if target.VCPU < current.VCPU {
+			direction = "down"
+		}
```

Comment:
I feel like a "getDirection" helper function would be nice

Reply:
Done